manim/mobject/graphing/coordinate_systems.py

```diff
@@ -1335,9 +1296,22 @@ def plot_derivative_graph(
     ) -> ParametricFunction:
         """Returns the curve of the derivative of the passed graph.
 
+        Parameters
+        ----------
+        graph
+            The graph for which the derivative will be found.
+        color
+            The color of the derivative curve.
+        **kwargs
```

Comment:
```suggestion
        kwargs
```

This seems inconsistent throughout the file. I think it should be changed.